config/permissions.yml

```diff
@@ -2319,6 +2319,12 @@ feature_user_list:
     expose: true
     label: 'fetch users from the API route.'
     loginRequired: true
+# Only needed for user list in one specific project
+feature_user_list_extended:
```

Comment:
should be add in bobhh/permissions.php under support role, If you add this permission, please add the link of bobhh branch in Description->linked branches

Reply:
Yes, I added it. You see the link below.